### PR TITLE
Make links in markdown note bolder

### PR DESCRIPTION
**Pull request by luka-nextcloud**
Signed-off-by: Luka Trovic <luka@nextcloud.com>

* Resolves: #3443
* Target version: master 

### Summary
Improve contrast link in Markdown note

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by stefan-niedermann** [comment]
Shouldn't this be done in the upstream `nextcloud-text` project?

**Comment by jancborchardt** [comment]
Regarding the screenshot at https://github.com/nextcloud/deck/issues/3443#issue-1061760621, what would be ideal is:
- The text part of the link (inside the [ ] brackets) in `var(--color-main-text)`
- Everything else (including the [ ] brackets) in `var(--color-text-maxcontrast)`

**Comment by juliushaertl** [comment]
> Shouldn't this be done in the upstream nextcloud-text project?

Deck is not yet using text



> * The text part of the link (inside the [ ] brackets) in `var(--color-main-text)`
> * Everything else (including the [ ] brackets) in `var(--color-text-maxcontrast)`

@luka-nextcloud Could you adjust that accordingly? Using CSS variables will have the benefit that the coolers will adjust properly when using the dark theme which can be toggled in the users accessibility settings.

File: src/components/card/Description.vue [hunk 1]
```diff
@@ -332,4 +332,20 @@ h5 {
...
+	color: var(--color-main-text);
+}
+
+.vue-easymde .cm-s-easymde .cm-formatting.cm-formatting-image.cm-image.cm-image-alt-text.cm-link{
+	color: var(--color-text-maxcontrast);
+}
+
+.vue-easymde .cm-s-easymde .cm-url {
+	color: #767676 !important;
+}
```
Comment (eneiluj) [inline src/components/card/Description.vue:350]: The image link color was ok but the basic link color wasn't.

@luka-nextcloud Are you ok with those changes?

```suggestion
.vue-easymde .cm-s-easymde .cm-link {
	color: var(--color-main-text);
}

.vue-easymde .cm-s-easymde .cm-string.cm-url,
.vue-easymde .cm-s-easymde .cm-formatting.cm-link,
.vue-easymde .cm-s-easymde .cm-formatting.cm-url,
.vue-easymde .cm-s-easymde .cm-formatting.cm-image {
	color: var(--color-text-maxcontrast);
}
```

**Review by eneiluj — CHANGES_REQUESTED** [review]

File: src/components/card/Description.vue (hunk 1 above)
Comment (luka-nextcloud) [reply src/components/card/Description.vue]: Ok, it looks better.

File: src/components/card/Description.vue [hunk 2]
```diff
@@ -334,4 +334,19 @@ h5 {
...
+.vue-easymde .cm-s-easymde .cm-string.cm-url,
+.vue-easymde .cm-s-easymde .cm-formatting.cm-link,
+.vue-easymde .cm-s-easymde .cm-formatting.cm-url,
+.vue-easymde .cm-s-easymde .cm-formatting.cm-image {
+	color: var(--color-text-maxcontrast);
+}
+
+.vue-easymde .cm-s-easymde .cm-url {
+	color: #767676 !important;
+}
```
Comment (eneiluj) [inline src/components/card/Description.vue:351]:
```suggestion
```
This is already done upper with `.vue-easymde .cm-s-easymde .cm-string.cm-url,`.

**Review by eneiluj — CHANGES_REQUESTED** [review]

**Review by eneiluj — APPROVED** [review]

**Pull request merged**